frame/collective/src/lib.rs

```diff
@@ -168,12 +175,129 @@ decl_error! {
 		AlreadyInitialized,
 		/// The close call is made too early, before the end of the voting.
 		TooEarly,
+		/// There can only be a maximum of `MaxMembers`.
+		TooManyMembers,
+		/// There can only be a maximum of `MaxProposals` active proposals.
+		TooManyProposals,
+		/// The given weight bound for the proposal was too low.
+		WrongProposalBound,
+	}
+}
+
+/// Functions for calcuating the weight of dispatchables.
+mod weight_for {
+	use frame_support::weights::{RuntimeDbWeight, Weight};
+
+	/// Calculate the weight for `set_members`.
```

Comment:
You should duplicate or move the raw weight information here next to these formulas.